tools-local/tasks/FinalizeBuild.cs

```diff
@@ -92,20 +92,13 @@ public override bool Execute()
 
                 try
                 {
-                    CopyBlobs($"{Channel}/Binaries/{SharedFrameworkNugetVersion}", $"{Channel}/Binaries/Latest/");
+                    CopyBlobs($"Runtime/{SharedFrameworkNugetVersion}", $"Runtime/Latest/");
 
-                    CopyBlobs($"{Channel}/Installers/{SharedFrameworkNugetVersion}", $"{Channel}/Installers/Latest/");
-
-                    CopyBlobs($"{Channel}/Installers/{SharedHostNugetVersion}", $"{Channel}/Installers/Latest/");
-
-                    // Generate the Sharedfx Version text files
-                    List<string> versionFiles = PublishRids.Select(p => $"{p.GetMetadata("VersionFileName")}.version").ToList();
+                    CopyBlobs($"Runtime/{SharedHostNugetVersion}", $"Runtime/Latest/");
```

Comment:
We have a lot of version usages problems in our build, and this really shows the problem.  I'll try to explain:

In a single build, we only ever build one version of things.  In our build code we call it `ProductVersion`.  It is the uber "I-don't-care-about-a-single-component" version.  Then each component can have its own version property because in servicing builds, the components are serviced separately.  We may only build a new SharedFramework, but not build a new host dotnet.exe.

So we have `SharedFrameworkNugetVersion` (the version of the Shared Framework we are building), `HostResolverVersion` (the version of the hostfxr.dll that should be used), `HostVersion` (the version of the dotnet.exe that should be used).

Let's say in the first servicing release, we just need to build a new Shared Framework.  So ProductVersion and SharedFrameworkVersion will be `2.0.1`, but HostVersion will still be `2.0.0` because we still want to use the same host that was shipped with 2.0 RTM.  Here you are going to be copying all the `2.0.0` blobs into `Runtime/Latest`, thus blowing away all the `2.0.1` blobs that were just copied above.

So 2 things should change here:

1. Don't use a single component's version during publish/finalize unless you are strictly referencing that single component.  Instead use `ProductVersion`.
1. Don't CopyBlobs twice.  Just do it once for ProductVersion.